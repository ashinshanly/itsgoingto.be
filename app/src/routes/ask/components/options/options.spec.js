/* global expect, jest */
import React from 'react'
import { shallow } from 'enzyme'
import Options from './options'

const props = {
  hasQuestion : true,
  poll        : { question: 'Question', answers: [] },
}

describe('(Route) Ask', () => {
  describe('(Component) Options', () => {
    describe('(Render)', () => {
      describe('when hasQuestion is true', () => {
        it('matches snapshot', () => {
          const wrapper = shallow(<Options {...props} hasQuestion={true} />)
          expect(wrapper).toMatchSnapshot()
        })
      })

      describe('when hasQuestion is false', () => {
        it('matches snapshot', () => {
          const wrapper = shallow(<Options {...props} hasQuestion={false} />)
          expect(wrapper).toMatchSnapshot()
        })
      })
    })
  })
})
