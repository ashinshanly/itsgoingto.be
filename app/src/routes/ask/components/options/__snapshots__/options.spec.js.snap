// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Route) Ask (Component) Options (Render) when hasQuestion is false matches snapshot 1`] = `
<div>
  <div
    className="options hideable gone"
  >
    <Connect(MultipleChoice)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
    <Connect(Passphrase)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
    <Connect(EndPoll)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
  </div>
</div>
`;

exports[`(Route) Ask (Component) Options (Render) when hasQuestion is true matches snapshot 1`] = `
<div>
  <div
    className="options hideable"
  >
    <Connect(MultipleChoice)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
    <Connect(Passphrase)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
    <Connect(EndPoll)
      poll={
        Object {
          "answers": Array [],
          "question": "Question",
        }
      }
    />
  </div>
</div>
`;
